libkode/printer.h

```diff
@@ -117,6 +119,16 @@ class KODE_EXPORT Printer
      */
     void printImplementation( const File &file, bool createHeaderInclude = true );
 
+    /**
+     * @brief printCodeIntoFile
+     * Writes the string passed through the code parameter to the file refernced
+     * by the file parameter in the case if it differs from the content of the file pointed by the
+     * file parameter.
+     * @param code reference to a Code object which contents needs to be printed
+     * @param file the target file in unopened state with filename set
+     */
+    void printCodeIntoFile (const Code &code, QFile *file);
```

Comment:
This is an internal helper method, it should be private.

Here as well, spaces inside parenthesis, but not before '('.